packages/shared/lib/core/network/utils/getOfficialExplorer.ts

```diff
@@ -0,0 +1,6 @@
+import { EXPLORER_URLS } from '../constants'
+import { NetworkProtocol, NetworkType } from '../enums'
+
+export function getOfficialExplorer(protocol: NetworkProtocol, type: NetworkType): string {
```

Comment:
I think we should rename this to `getOfficialExplorerUrl`.

Reply:
Done